deny.toml

```diff
@@ -103,61 +92,37 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
```

Comment:
Keep this, no license basically means [we can't use it.](https://choosealicense.com/no-permission/)